docs/api/web-contents.md

```diff
@@ -1519,12 +1519,16 @@ console.log(requestId)
 * `opts` Object (optional)
   * `stayHidden` boolean (optional) -  Keep the page hidden instead of visible. Default is `false`.
   * `stayAwake` boolean (optional) -  Keep the system awake instead of allowing it to sleep. Default is `false`.
+  * `outputSize` Object (optional)
```

Comment:
Same as above - units need specification!

Reply:
added: _The desired width and height in pixels (positive numbers) of the returned image_